### ISSUE TITLE
Introduce constant or getter to get "cnf" claim

### ISSUE THREAD

**Issue by nenaraab** [SAP/cloud-security-xsuaa-integration#508]

**Comment by liga-oz** [comment]
Have it in this PR https://github.com/SAP/cloud-security-xsuaa-integration/pull/490

**Comment by nenaraab** [comment]
@liga-oz yes, I think we leverage a lot from #490 (but without the cnf validation) as prerequisite for SECAUTH-1223

**Comment by nenaraab** [comment]
as the constant is kept private we should provide a getter for now in context of SapIdToken
